### PR TITLE
feat: remove link and add description for unenrollable runs in more dates

### PR DESCRIPTION
**Pull request by asadali145**
# What are the relevant tickets?

Fixes https://github.com/mitodl/hq/issues/1647

# Description (What does it do?)

Updates more dates popup. Removes the link for the future course runs(enrollment date in future). Also, display the enrollment date for such runs.

# Screenshots (if appropriate):

- [x] Desktop screenshots
- [ ] Mobile width screenshots
Screen Shot 2023-06-22 at 5 06 24 PM

# How can this be tested?

Create 2-course runs. One with an enrollment date in the past and one with an enrollment start date in the future. Visit the course detail page. Verfiy that the more dates pop up has 2 runs, 1 is a clickable link and other one is just date with enrollment start date as well.

# Additional Context

### CODE REVIEW

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍 

Could you get approval from @pdpinch on the screenshot?

**Comment by asadali145** [comment]
@pdpinch Could you please have a look at the attached SS for the future course runs? Does it look good to you?

**Comment by pdpinch** [comment]
Bilal is working on a redesign of this page, so this isn't worth a major effort. 

But, the alignment and spacing looks really weird. I don't know why the first line is indented. 

Would it be simpler to make it a list of two items?

**Comment by pdpinch** [comment]
@asadali145 what's holding this up?

**Comment by pdpinch** [comment]
@arslanashraf7 please wrap this up in Asad's absence. I'd like to clean up the formatting, but that's not as important as removing the misleading links.

**Comment by arslanashraf7** [comment]
@pdpinch 

>But, the alignment and spacing looks really weird. I don't know why the first line is indented.

The original/previous design has the indentation, I've rebased this PR and adjusted the style for the disabled links in https://github.com/mitodl/mitxonline/pull/1684/commits/6cca0b1aecd8de7cd32cb35e8002df514eccb8c3. A new screenshot is attached below, Please take a look and let me know if this looks better now.

Also, Since we are going to change the design in the future anyway, I've not made any major changes to this.

image

**Comment by arslanashraf7** [comment]
@pdpinch 
>Would it be simpler to make it a list of two items?

I didn't completely get this, but I tried something. Please take a look at the screenshot below. Note that, this change is not committed, I stashed it locally. If you decide to go with this style instead of the one in the above comment, I'll commit these changes.

image

**Comment by pdpinch** [comment]
@arslanashraf7 looks great!

**Comment by arslanashraf7** [comment]
> @arslanashraf7 looks great!

@pdpinch Just to reconfirm, I'm assuming you are talking about https://github.com/mitodl/mitxonline/pull/1684#issuecomment-1632215768?

**Comment by pdpinch** [comment]
> Just to reconfirm, I'm assuming you are talking about https://github.com/mitodl/mitxonline/pull/1684#issuecomment-1632215768?

Sorry, yes, the design in https://github.com/mitodl/mitxonline/pull/1684#issuecomment-1632239969 looks great and we should move ahead.

**Pull request merged**
